README.md

```diff
@@ -311,21 +349,34 @@ callback let you add assertions and enrich the test.
     message is caracterized by its *api* and the message level *error*,
     *warning*, *notice*, *info* or *debug*.
 
-    The timeout argument is in seconds and could be omitted if you doesn't need
-    to wait to receive the event.
+* **_AFT.testEvtNotReceived(testName, eventName)**
```

Comment:
Add a timeout event as last arguments even if not implemented right now, it shouldn't be long. So for one time we have the documentation more advanced than the code...